lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
```

Comment:
I think it makes sense and since the output directory can be modified the manifest can remain same 👍  I was thinking about the same for `config` option - cab't think of a scenario this will need changing in any app.